src/index.html

```diff
@@ -6,9 +6,44 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img class="header-logo" src="images/logo.png" alt="logo">
+      </a>
+      <nav class="header-navigation">
+        <ul class="header-navigation_items">
+          <li class="header-navigation_item is-active">
```

Comment:
BEM uses double underscores to denote elements.
```suggestion
      <nav class="header-navigation">
        <ul class="header-navigation__items">
          <li class="header-navigation__item header-navigation__item--active">
```